Docker docs update (#177)
whamcloud/Online-Help: docs/Contributor_Docs/cd_Running_IML_On_Docker_With_Vagrant_Servers.md

# Running IML On Docker With Vagrant Servers

[**Software Contributor Documentation Table of Contents**](cd_TOC.md)

## Overview

> IML is capable of running in a docker swarm (using docker stack) where each services is run in its own container. This approach allows IML to be deployed quickly onto the host node. The lustre servers however, need to be run on hardware or in a virtual machine. During development, it is common to deploy IML in a docker swarm and use virtuabox instances as servers. Setting up network communication between the docker swarm and the virtualbox nodes can be a bit tricky. This document describes the process of deploying IML to a docker swarm and how to connect it to virtualbox vm's when adding server nodes.

Reviewer: ```suggestion
IML is capable of running in a docker swarm (using docker stack) where each manager service is run in its own container. This approach allows IML to be deployed quickly onto the host node. The lustre servers however, need to be run on hardware or in a virtual machine. During development, it is common to deploy IML in a docker swarm and use virtuabox instances as servers. Setting up network communication between the docker swarm and the virtualbox nodes can be a bit tricky. This document describes the process of deploying IML to a docker swarm and how to connect it to virtualbox vm's when adding server nodes.
```

> IML is capable of running in a docker swarm (using docker stack) where each services is run in its own container. This approach allows IML to be deployed quickly onto the host node. The lustre servers however, need to be run on hardware or in a virtual machine. During development, it is common to deploy IML in a docker swarm and use virtuabox instances as servers. Setting up network communication between the docker swarm and the virtualbox nodes can be a bit tricky. This document describes the process of deploying IML to a docker swarm and how to connect it to virtualbox vm's when adding server nodes.

Reviewer: ```suggestion
IML is capable of running in a docker swarm (using docker stack) where each services is run in its own container. This approach allows IML to be deployed quickly onto the host node. The lustre servers however, need to be somewhere with a full kernel. During development, it is common to deploy IML in a docker swarm and use virtuabox instances as servers. Setting up network communication between the docker swarm and the virtualbox nodes can be a bit tricky. This document describes the process of deploying IML to a docker swarm and how to connect it to virtualbox vm's when adding server nodes.
```
